<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="1" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="academy.tests.SubmitOrderTest">
        <methods>
          <include name="tearDown"/>
          <include name="launchApplication"/>
          <include name="submitOrder"/>
          <include name="OrderHistoryTest" invocation-numbers="0"/>
        </methods>
      </class> <!-- academy.tests.SubmitOrderTest -->
      <class name="academy.tests.ErrorValidationTest">
        <methods>
          <include name="IncorrectCredentialValidation"/>
          <include name="tearDown"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- academy.tests.ErrorValidationTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
